packages/nimble-components/src/table-column/number-text/tests/table-column-number-text.spec.ts

```diff
@@ -31,7 +34,10 @@ async function setup(source: ColumnInstances): Promise<Fixture<Table<SimpleTable
                     Column 2
                 </${tableColumnNumberTextTag}>
             </nimble-table>`,
-        { source }
+        {
+            source,
+            parent: themeProvider
```

Comment:
I see this was a pattern introduced in date column for locale.

Could we instead include `nimble-theme-provider` in the template instead of imperatively constructing it and making it the parent?

I'm kinda concerned that it bypasses the fixture workaround we put here:

https://github.com/ni/nimble/blame/ac760e231e1a9b39e898c8753a0b4c95f7e0b41a/packages/nimble-components/src/utilities/tests/fixture.ts#L152-L157

table is viewport sensitive-ish so not sure if this could cause issues by the table being pushed out the viewport or not.

Maybe we should go as far as removing that as an option from the fixture helper..

Reply:
I've updated these number-text column tests to include the theme-provider in the template.

I've we're happy with the way this looks, I can make a separate PR to apply the same change to the date-text column and figure out the next step towards getting rid of the `parent` option (either create tech debt issue or just do it).

Reply:
LGTM!